app-admin/amazon-ec2-init/amazon-ec2-init-20101127-r1.ebuild

```diff
@@ -1,7 +1,7 @@
-# Copyright 1999-2021 Gentoo Authors
+# Copyright 1999-2023 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI="6"
+EAPI=8
```

Comment:
Drop blank DEPEND, IUSE, SRC_URI, and put S directly below HOMEPAGE per skel.ebuild order